third_party/move/move-compiler/transactional-tests/tests/control_flow/break_continue_for_loop.move

```diff
@@ -2,11 +2,13 @@
 script {
     fun main() {
         let x = 0;
-        for (i in 0..10) {
-            if (x >= 5) break;
-            x = x + 1;
-            continue
-        }
-        assert!(move x == 5, 42);
+        for (i in 0..10) { // 0,  1,  2,  3,  4, 5, 6, 7, 8, 9, 10
+            x = x + 1;     // 1,  4, 13, 16,
+            if (x >= 15) break;
+	    x = x + 2;     // 3,  6, 15,  -
```

Comment:
```suggestion
	        x = x + 2;     // 3,  6, 15,  -
```

Reply:
Indentation here and below (assignments to `x`) should be fixed for readability.

Reply:
oops, emacs added tabs.  gotta get rid of that.